### PR TITLE
[VFX]Rollback Top level menus

### PR DESCRIPTION
**Pull request by alex-vazquez**
---
### Purpose of this PR
Rollback VFX to Visual Effects as discussed.

---
### Testing status
Check that everything is back to what it was.

### CODE REVIEW

File: com.unity.visualeffectgraph/Editor/GraphView/VFXViewWindow.cs [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+using UnityEditor.VFX;
+using UnityEngine.UIElements;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityObject = UnityEngine.Object;
+using System.IO;
+using UnityEditor.VersionControl;
 
-namespace UnityEditor.VFX.UI
+namespace  UnityEditor.VFX.UI
```
Comment (julienf-unity) [inline com.unity.visualeffectgraph/Editor/GraphView/VFXViewWindow.cs:16]: Strange the formatter did not catch this? @theo-pnv

File: com.unity.visualeffectgraph/Editor/Utilities/pCache/BakeTool/PointCacheBakeTool.cs [hunk 2]
```diff
@@ -1,10 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
```
Comment (julienf-unity) [inline com.unity.visualeffectgraph/Editor/Utilities/pCache/BakeTool/PointCacheBakeTool.cs:2]: I guess those includes were unused ones that were removed by your first PR and have been therefore rolled back by your second?

**Review by julienf-unity — APPROVED** [review]

**Comment by alex-vazquez** [comment]
@VladNeykov could you please do a quick test that all the menus are correct as they were before? Thank you :)

**Review by VladNeykov — APPROVED** [review]
LGTM

**Pull request merged**